pkg/provisioner/core/provisioner.go

```diff
@@ -87,8 +87,8 @@ func NewOCIProvisioner(kubeClient kubernetes.Interface, nodeInformer informersv1
 		nodeLister:       nodeInformer.Lister(),
 		nodeListerSynced: nodeInformer.Informer().HasSynced,
 		storageClassProvisioners: map[string]plugin.ProvisionerPlugin{
-			"oci":      blockProvisioner,
-			"oci-ext3": blockProvisioner,
```

Comment:
Dispatching here should be done on some additional parameter e.g. type: block, type: nfs. This allows us to support FSS and Block. The block provisioner gets the volume options passed in so it can lookup the fsType directly without it being passed in.

Reply:
We discussed this as an option yesterday. Personally I prefer the symlink approach but this to overloading `fsType` also.